api/transactions/transactions_test.go

```diff
@@ -74,6 +79,16 @@ func TestTransaction(t *testing.T) {
 	} {
 		t.Run(name, tt)
 	}
+
+	// Call transaction
+	for name, tt := range map[string]func(*testing.T){
```

Comment:
Can another test be added here that does the same checks, or a summarized version, of the checks that the mempool does to verify tx integrity ?

Reply:
Ah! These seem to be covered already yeah ?